LibGit2/Functions/New-GitTag.ps1

```diff
@@ -0,0 +1,95 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function New-GitTag{
+    <#
+    .SYNOPSIS
+    Creates a new tag.
+    
+    .DESCRIPTION
+    Can create a new tag, or update the target of an existing tag.
+
+    This function implements the `git tag <tagname> <target>` command.
+    
+    .EXAMPLE
+    New-GitTag -Name 'BranchBaseline'
+    
+    Creates a new tag, `BranchBaseline`, for the HEAD of the current directory.
+    
+    .EXAMPLE
+    New-GitTag -Name 'BranchBaseline' -Target 'branch'
+    
+    Demonstrates how to create a tag pointing to the head of a branch.
+    
+    .EXAMPLE
+    New-GitTag -Name 'BranchBaseline' -Force
+    
+    Demonstrates how to change the target a tag points to, to the current HEAD.
+    #>
+
+    [CmdletBinding()]
+    param(
+        [string]
+        # Specifies which git repository to add the tag to. Defaults to the current directory.
+        $RepoRoot = (Get-Location).ProviderPath,
+
+        [Parameter(Mandatory=$true)]
+        [string]
+        # The name of the tag.
+        $Name,
+
+        [string]
+        # The commit this tag should point at. Defaults to "HEAD".
+        $Target = "HEAD",

```

Comment:
Change this parameter name to "Revision". Its doc comment should be: "The revision the tag should point to/reference. A revision can be a specific commit ID/sha (short or long), branch name, tag name, etc. Run git help gitrevisions or go to https://git-scm.com/docs/gitrevisions for full documentation on Git's revision syntax."